manifests/prometheus/alerts.d/bosh-stemcells-not-in-sync.yml

```
@@ -7,7 +7,7 @@
     name: BoshStemcellsNotInSync
     rules:
       - alert: BoshStemcellsNotInSync
-        expr: "sum by (bosh_stemcell_version) (bosh_deployment_stemcell_info) != 3"
+        expr: "count (count by (bosh_stemcell_version) (bosh_deployment_stemcell_info)) == 1"
```

```suggestion
        expr: "count (count by (bosh_stemcell_version) (bosh_deployment_stemcell_info)) != 1"
```

Sorry my suggestion contained an error, we want to be alerted when the stemcells are not in sync, ie when count != 1

[The end of july in London Grafana triggers this alert](https://grafana-1.london.cloud.service.gov.uk/explore?left=%5B%22now-30d%22,%22now%22,%22prometheus%22,%7B%22expr%22:%22count%20(count%20by%20(bosh_stemcell_version)%20(bosh_deployment_stemcell_info))%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D)

---

Fixed. Oops. Approval please :)